packages/react/src/hooks/useCogsMediaConfig.ts

```diff
@@ -2,7 +2,7 @@ import { CogsConnection, MediaClientConfig } from '@clockworkdog/cogs-client';
 import { useEffect, useState } from 'react';
 
 export default function useCogsMediaConfig<
-  MediaConfigExtra extends unknown = Record<never, never>,
+  MediaConfigExtra,
```

Comment:
Do we need to keep the default type here so you can use it without passing anything? How about we're more specific?

```ts
export default function useCogsMediaConfig<
  MediaConfigExtra extends Record<string, unknown> = Record<string, never>
>
```

Reply:
I removed this off the back of a linting suggestion

Reply:
Ah my mistake, Typescript will default it to `unknown` if not specified. I thought it would force you to set a type.